js/components/non_interoperable_info.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+function NonInteroperableInfo({ agencyComponent }) {
+  const submissionUrl = agencyComponent.submission_web && agencyComponent.submission_web.uri;
+  const submissionInstructions = /^https?:/.test(submissionUrl) ?
```

Comment:
What does `test` do?

Reply:
It applies tests the string against the regex but instead of returning a match object, it just returns a boolean, true if it matches, false otherwise.